### ISSUE TITLE
WCAG 2.2 - Bounding box anchor in 2.5.8 Target Size (Minimum) is missing definition - Editorial

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/wcag#3491]
In the Understanding of SC [2.5.8 Target Size (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html#dfn-pointer-input) the "bounding box" anchor link is not working and the "minimum bounding box" definition is not present within the page.

Link: https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html

I've created a pull request adding "bounding box" to the data-lt attribute.
